code/Properties/launchSettings.json

```diff
@@ -1,9 +1,9 @@
 {
-    "profiles": {
-        "ttt-reborn": {
-            "commandName": "Executable",
-            "executablePath": "C:\\Program Files (x86)\\Steam\\steamapps\\common\\sbox\\sbox.exe",
-            "commandLineArgs": "+developer +sw -noassert -tools"
-        }
+  "profiles": {
+    "ttt-reborn": {
+      "commandName": "Executable",
+      "executablePath": "C:\\Program Files (x86)\\Steam\\steamapps\\common\\sbox\\sbox.exe",
+      "commandLineArgs": "\u002Bdeveloper \u002Bsw -noassert -tools"
     }
+  }
```

Comment:
is there a reason you switched back to 2 space indentation here?

Reply:
No reason, IDE messing with me.

Reply:
Agh, my IDE for whatever reason is continuing to mess with this file. Anyways, should we just add this file to our `gitignore`? Each of us will have a unique `executablePath`.

Reply:
Yea guess it's the generation tool / csproj. Anyways, doesn't matter. I agree, please add this file to the gitignore, so it is still in this repo but won't update in the upcoming time :)

Reply:
that is not how .gitignore works ^^ once a file is tracked by git, the .gitignore does nothing.

That file would need to be explicitly removed **and** put into the .gitignore